frontend/src/stories/components/HelpRequest/HelpRequestForm.stories.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import HelpRequestForm from "main/components/HelpRequest/HelpRequestForm"
+import { helpRequestFixtures } from 'fixtures/helpRequestFixtures';
+
+export default {
+    title: 'components/HelpRequests/HelpRequestForm',
+    component: HelpRequestForm
+};
+
+const Template = (args) => {
+    return (
+        <HelpRequestForm {...args} />
+    )
+};
+
+export const Create = Template.bind({});
+
+Create.args = {
+    buttonLabel: "Create",
+    submitAction: (data) => {
+         console.log("Submit was clicked with data: ", data); 
+         window.alert("Submit was clicked with data: " + JSON.stringify(data));
+    }
+};
+
+export const Update = Template.bind({});
+
+Update.args = {
+    initialContents: helpRequestFixtures.oneHelpRequest,
```

Comment:
Either change to `oneHelpRequest[0]` because it is stored as a list in `helpRequestFixtures.js` or change `helpRequestFixtures.js` to not store `oneHelpRequest` as a list. Currently doesn't show up in storybook correctly.
![image](https://github.com/ucsb-cs156-f23/team03-f23-7pm-2/assets/72269643/c9f22e0d-4877-4e1d-aac3-6d539e0b9120)

Reply:
Ty for catching this bug. Changed line 29 to oneHelpRequest[0] in the most recent commit.